Adding a release step to docs (#6480)
elastic/eui: wiki/releasing-versions.md

  - if you have 2FA enabled, you may be prompted for an [OTP or other token](https://github.com/settings/tokens)

### Releasing `@elastic/eui`

> Check for latest code and dependencies. Run the following commands in a new terminal window:

Reviewer: Minor copy suggestion
```suggestion
Ensure you have both the latest code and dependencies by running the following commands:
```

> 
> ```shell
> git checkout main && git pull
> 
> # Run after the git pull completes
> yarn

Reviewer: This can be simplified some
```suggestion
```sh
git checkout main && git pull && yarn
```
